api/src/main/java/com/datastrato/gravitino/SupportsSchemas.java

```diff
@@ -39,12 +39,12 @@ public interface SupportsSchemas {
    *
    * <p>If an entity such as a table, view exists, its parent schemas must also exist and must be
    * returned by this discovery method. For example, if table a.b.t exists, this method invoked as
-   * listSchemas(a) must return [a.b] in the result array
+   * listSchemas(a) must return [b] in the result array
    *
-   * @return An array of schema identifier under the namespace.
+   * @return An array of schema names under the namespace.
```

Comment:
The comments in the implementation class also need to be modified：
https://github.com/datastrato/gravitino/blob/c8dabb745e7cb7b27573089b85af96c12a71e45a/clients/client-java/src/main/java/com/datastrato/gravitino/client/BaseSchemaCatalog.java#L70

Reply:
Thx! I just updated the java doc in BaseSchemaCatalog.java, please check;